src/components/inputs/button/button.stories.tsx

```diff
@@ -201,3 +207,35 @@ export const IconButtons: ComponentStory<typeof Button> = () => {
     </div>
   );
 };
+
+/**
+ * File upload buttons used with or without icons.
+ */
+export const UploadButton: ComponentStory<typeof Button> = () => {
+  return (
+    <div className="story__button">
+      <input
+        accept="image/*"
+        id="icon-button-file"
+        multiple
+        type="file"
+        style={{ display: 'none' }}
+      />
+      <label htmlFor="icon-button-file">
+        <IconButton icon={<PlusCircle />} component="span" />
+      </label>
+      <input
+        accept="image/*"
+        id="button-file"
+        multiple
+        type="file"
+        style={{ display: 'none' }}
+      />
+      <label htmlFor="button-file">
+        <Button variant="contained" color="primary" component="span">
+          Upload
+        </Button>
+      </label>
+    </div>
+  );
+};
```

Comment:
Some minors here:

- There is no indication where each button starts and finish, it will be difficult for the developer to understand what he/she needs to copy and paste. Maybe a comment or something?
- The buttons don't look nice in the story, they are too close to each other. The class `story__button` that you are using, does not do what you think it does.